101-databricks-all-in-one-template-for-vnet-injection/azuredeploy.json

```diff
@@ -285,7 +111,13 @@
               "addressPrefix": "[parameters('publicSubnetCidr')]",
               "networkSecurityGroup": {
                 "id": "[variables('nsgId')]"
-              }
+              },
+              "delegations": [{
+                "name": "[concat('databricks-del-', uniqueString(parameters('publicSubnetName')))]",
```

Comment:
does this need to be globally unique?  if not use a more meaningful name (e.g. databrick-del-public)

Reply:
It doesn't needs to be globally unique. It needs to be unique for subnet. So, "databrick-del-public" makes sense. I will update.